cluster/tool/auth-cli/main.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
why do we have an `auth-cli` in this repo?

Reply:
good catch. I forgot to remove it as added it for testing.